orchestra/static/orchestra/timing/timecard/controller.js

```diff
@@ -52,20 +52,14 @@
         return {
           description: entry.description,
           timeWorked: entry.time_worked.componentize(),
-          assignment: orchestraTasks.tasksById[entry.assignment]
+          task: orchestraTasks.tasksByAssignmentId[entry.assignment]
```

Comment:
Yeah, but only if the client opens up the edit view before the tasks have loaded from the server—I think this is probably a pretty small case, but I'll fix it to be sure.